install_config/registry/deploy_registry_existing_clusters.adoc

```diff
@@ -380,6 +380,34 @@ MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCyOJ5garOYw0sm
 ...
 -----END PRIVATE KEY-----
 ----
+
+- If using Subject Alternative Names (SAN), the server certificate should contain the following in the SAN list:
```

Comment:
Remove "If using Subject Alternative Names (SAN)". 

The secured registry should contain the following in the SAN list